lua/gruvbox.lua

```diff
@@ -360,9 +361,9 @@ local function get_groups()
     PmenuSel = { fg = colors.bg2, bg = colors.blue, bold = config.bold },
     PmenuSbar = { bg = colors.bg2 },
     PmenuThumb = { bg = colors.bg4 },
-    DiffDelete = { bg = colors.dark_red },
-    DiffAdd = { bg = colors.dark_green },
-    DiffChange = { bg = colors.dark_aqua },
+    DiffDelete = config.invert_diff and { fg = colors.bg0, bg = colors.red } or { bg = colors.dark_red },
+    DiffAdd = config.invert_diff and { fg = colors.bg0, bg = colors.green } or { bg = colors.dark_green },
+    DiffChange = config.invert_diff and { fg = colors.bg0, bg = colors.aqua } or { bg = colors.dark_aqua },
```

Comment:
Style not quite a revert of the style, if the user has `config.inverse = true`.

```suggestion
    DiffDelete = config.invert_diff and { fg = colors.bg0, bg = colors.red, reverse = config.inverse } or { bg = colors.dark_red },
    DiffAdd = config.invert_diff and { fg = colors.bg0, bg = colors.green, reverse = config.inverse } or { bg = colors.dark_green },
    DiffChange = config.invert_diff and { fg = colors.bg0, bg = colors.aqua, reverse = config.inverse } or { bg = colors.dark_aqua },
```

Reply:
Mmh, I think there are actually two things happening in the new configuration:

* The colors changed.
* And the `reverse` is unconditionally false.

I want _both_ the old colours _and_ `reverse = true`.

Whereas you only want the old colours?

Reply:
You are right